manifest.json

```diff
@@ -43,7 +43,7 @@
   },
   "proxy": {
     "whitelist": [{
-      "url": "https://example.com/.*",
+      "url": "https://gitlab.com/.*",
```

Comment:
Settings can now be injected into this url pattern, see https://support.deskpro.com/en-US/guides/developers/app-proxy#undefined-1

E.g.

```
  "url": "https://__instance_hostname_/.*"
```

Reply:
I'll configure it in this story https://app.shortcut.com/deskpro/story/93951/gitlab-write-setup-guide-and-description